src/index.html

```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#logo" class="logo"><img src="images/logo.png" alt="logotip"></a>
+      <nav class="box">
```

Comment:
try to give more meaningful names - for this one you can apply .nav, then .nav__list and .nav__item